pulseapi/profiles/test_views.py

```diff
@@ -105,3 +105,58 @@ def test_program_year_uniqueness(self):
         # as found in the bootstrap migration:
         (profile, created) = ProgramYear.objects.get_or_create(value='2018')
         self.assertEqual(created, False)
+
+    def test_profile_searching(self):
+        profile_types = ['a', 'b', 'c']
+        program_types = ['a', 'b']
+        program_years = ['a', 'b']
+
+        for v1 in profile_types:
```

Comment:
any reason we're not doing this in the bootstrap logic?

Reply:
I wanted this test to verify the functionality irrespective of prepopulated data.